extensions/elytron-security-properties-file/runtime/src/main/java/io/quarkus/elytron/security/runtime/ElytronPropertiesFileRecorder.java

```diff
@@ -147,7 +147,7 @@ public void run() {
                                     .generatePassword(new DigestPasswordSpec(user, config.realmName, hashed));
                         } catch (Exception e) {
                             throw new RuntimeException("Unable to register password for user:" + user
-                                    + " make sure it is a valid hex encoded MD5 hash", e);
+                                    + " make sure it is a valid hex encoded " + runtimeConfig.algorithm.getName().substring(7).toUpperCase() + " hash", e);
```

Comment:
Are we sure about the algorithm name being always more than 7 characters long?

Reply:
The Algorithm names are based on the static strings from
 https://wildfly-security.github.io/wildfly-elytron/documentation/api/eap74/org/wildfly/security/password/interfaces/DigestPassword.html

The names all start with `digest-` - my intention with substring(7) was to remove the prefix to only display the real algorithm name. When WildFly changes these static strings, it will break. We can use `.replace("digest-", "")` instead of course or display the original name which WildFly gives us including the `digest-` prefix.

Reply:
@tamaro-skaljic yeah, I think I would play if safe and display the string as is.

Also make sure you run the formatter with `./mvnw process-sources` and squash your commits.

Thanks!